blocks/liveby-schools/liveby-schools.css

```diff
@@ -0,0 +1,163 @@
+main .section.liveby-schools-container {
+  max-width: 1150px;
+  margin: 0 auto;
+}
+
+main .section.liveby-schools-container p {
+  max-width: 900px;
+  margin: 2em auto;
+}
+
+.liveby-schools .sort {
+  display: flex;
+  width: 100%;
+  margin: 8px 0;
+  padding-left: 15px;
+  padding-right: 15px;
+  font-family: var(--font-family-proxima);
+  font-size: var(--body-font-size-m);
+  font-weight: 400;
+  justify-content: flex-end;
+}
+
+.liveby-schools .sort select {
+  appearance: none;
+  padding: 8px 32px 8px 8px;
+  color: var(--body-color);
+  font-family: var(--font-family-proxima);
+  font-size: var(--body-font-size-m);
+  font-weight: 400;
+  border-color: var(--gray);
+  border-radius: 2px;
+  background: url("https://pages.liveby.com/static/media/triangle.f2d6139e.svg") 93% center no-repeat rgb(255 255 255);
+}
+
+.liveby-schools .school-lists {
+  display: flex;
+  width: 100%;
+  flex-flow: column;
+}
+
+.liveby-schools .school-lists section {
+  margin: 1em 0;
+}
+
+.liveby-schools .actions a {
+  padding: .85em 1.15em;
+  font-size: var(--body-font-size-m);
+  color: var(--white);
+  background-color: var(--primary-color);
+}
+
+.liveby-schools .school-lists section h3 {
+  margin-bottom: 1em;
+  padding-top: 0;
+  padding-bottom: .5em;
+  font-family: var(--font-family-georgia);
+  font-weight: 700;
+  font-size: var(--heading-font-size-s);
+  border-bottom: 1px solid var(--border-gray);
+}
+
+.liveby-schools .school-lists section .school-item {
+  display: flex;
+  margin-bottom: 1.5em;
+}
+
+.liveby-schools .school-lists section .school-item:nth-of-type(n+5) {
+  display: none;
+}
+
+.liveby-schools .school-lists.show-all section .school-item:nth-of-type(n+5) {
+  display: flex;
+}
+
+.liveby-schools .school-lists section .school-item .school-details {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  flex: 1 1;
```

Comment:
i think you can use  flex shorthand to include flex-basis as well. see https://www.w3schools.com/cssref/css3_pr_flex.php